core/gui/package.json

```diff
@@ -58,6 +58,7 @@
     "jszip": "3.10.1",
     "lodash-es": "4.17.21",
     "marked": "4.3.0",
+    "monaco-breakpoints": "0.1.2",
```

Comment:
What's the reason to use `0.1.2` instead of the latest `0.2.0`?

Reply:
updated to 0.2.0